demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -114,6 +115,7 @@ function ProductsGridToolbar({ toolbarAction, exportApi }: { toolbarAction?: Rea
                             onClick: () => exportApi.exportGrid(),
                             disabled: exportApi.loading,
                         },
+                        <PublishAllProducts key="publish" />,
```

Comment:
Currently, the Excel export will always be the first item in the overall actions. Is this okay? If not, how would we change this?

Reply:
for now, it is ok.

we could probably specify a custom order like that:
```
        overallActions: [
            {
                name: "publish",
                .....
            },
            {
                type: "excelExport" //or name?
            }
```

and if excelExport isn't in the array we use the default position.

Reply:
I've created a task for this: https://vivid-planet.atlassian.net/browse/COM-1632

We could also revise the excelExport option to always be an element in the overallActions